src/calendar/calendar.tsx

```diff
@@ -1,6 +1,21 @@
+import styled from "styled-components";
 import { InternalCalendar } from "../shared/internal-calendar";
+import { Color } from "../color";
 import { CalendarProps } from "./types";
 
-export const Calendar = (props: CalendarProps) => {
-    return <InternalCalendar type="standalone" {...props} />;
+export const Calendar = ({ className, ...otherProps }: CalendarProps) => {
+    return (
+        <Wrapper className={className}>
+            <InternalCalendar type="standalone" {...otherProps} />
+        </Wrapper>
+    );
 };
+
+// =============================================================================
+// STYLING
+// =============================================================================
+const Wrapper = styled.div`
+    width: 41rem;
+    border: 1px solid ${Color.Neutral[5]};
+    border-radius: 8px;
```

Comment:
```suggestion
    border-radius: 0.75rem;
```

Reply:
Usually for border radius, we use absolute values to prevent any discrepancies

Reply:
ah ok, I thought that convention usually applies to small values like 1-3px only. but yes make sense to preserve border radius